### PR TITLE
Added countries test file, and run for Japan and Germany

### PR DESCRIPTION
**Pull request by bltemrah**

### CODE REVIEW

File: src/main/java/org/example/countries.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Countries
+
+1. Japan
+2. Germany
+3. China
+4. Korea
+3. Italy
+4. Usa
+
```
Comment (tscselma) [inline src/main/java/org/example/countries.txt:9]: @bltemrah  there seems a problem on the ordering numbers

File: src/main/java/org/example/countries.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Countries
+
+1. Japan
+2. Germany
```
Comment (fahriozcan) [inline src/main/java/org/example/countries.txt:4]: Good

File: src/main/java/org/example/countries.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+3. China
+4. Korea
```
Comment (lalesi85) [inline src/main/java/org/example/countries.txt:6]: My fav one

**Comment by yuyi2000101** [comment]
Nice job

**Review by fahriozcan — APPROVED** [review]

File: src/main/java/org/example/countries.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+3. Italy
```
Comment (Emretester) [inline src/main/java/org/example/countries.txt:7]: You should arrange the numbers

**Review by betulperco — APPROVED** [review]

File: src/main/java/org/example/countries.txt
Comment (sercankirac) [inline src/main/java/org/example/countries.txt:1]: looks good

**Comment by eyup1312** [comment]
Seems good

**Review by tscselma — APPROVED** [review]
ok now..

File: src/main/java/org/example/countries.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Countries
+
+1. Japan
+2. Germany
+3. China
```
Comment (bltemrah) [inline src/main/java/org/example/countries.txt:5]: @bltemrah d;asksdbaslkdbasjdkbsad

**Review by eyup1312 — COMMENTED** [review]
good job

**Pull request merged**